zebrad/src/components/sync.rs

```diff
@@ -351,7 +351,7 @@ where
             })
             .map_err(|e| eyre!(e))?;
 
-        tracing::debug!(?block_locator, "trying to obtain new chain tips");
+        tracing::info!(?block_locator, "trying to obtain new chain tips");
```

Comment:
The block locator is too big for a routine log, and we already log it elsewhere
```suggestion
        tracing::info!("trying to obtain new chain tips");
        tracing::debug!(?block_locator, "got block locator");
```